src/torchmetrics/functional/image/lpips.py

```diff
@@ -86,13 +85,21 @@ def __init__(self, requires_grad: bool = False, pretrained: bool = True) -> None
 
     def forward(self, x: Tensor) -> NamedTuple:
         """Process input."""
-        squeeze_output = namedtuple("squeeze_output", ["relu1", "relu2", "relu3", "relu4", "relu5", "relu6", "relu7"])
+
+        class SqueezeOutput(NamedTuple):
```

Comment:
```suggestion
        class _SqueezeOutput(NamedTuple):
```
Let's keep them protected so we do not need to deprecate if needed